### ISSUE TITLE
tensorflow 还支持另外一种batch normalization 的方法

### ISSUE THREAD

**Issue by YingxiaoKong** [MorvanZhou/tutorials#87]
莫凡你好，

我看到还有另外一种方法可以做batch normalization: tf.;ayers.batch_normalization. 这种方法不需要自己来算均值和方差。您有没有比较过这两种方法呢？

**Comment by MorvanZhou** [comment]
你好，请尽量使用新的方法，我当时做视频的时候，这个tf.layers.batch_normalization 功能还不存在。

**Comment by YingxiaoKong** [comment]
谢谢！

在 2020年4月24日，上午2:57，Morvan <notifications@github.com> 写道：

﻿

Closed #87<https://nam04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMorvanZhou%2Ftutorials%2Fissues%2F87&data=02%7C01%7Cyingxiao.kong%40vanderbilt.edu%7C7b1e083d4b4643b21f2608d7e8251728%7Cba5a7f39e3be4ab3b45067fa80faecad%7C0%7C0%7C637233118316206002&sdata=rk%2FDukKp7eNX0PRGLjHrJr8f%2BDv4obn%2BcJLlBEeCbtE%3D&reserved=0>.
